### PR TITLE
KEYCLOAK-19853 Update Arquillian version

### PR DESCRIPTION
**Pull request by Aboullos**
Update Arquillian to last version to support Edge testing

### CODE REVIEW

**Review by stianst — APPROVED** [review]
LGTM Although there was some issues with the model tests. Unsure if this could be caused by this PR, but I've restarted the jobs.

**Comment by Aboullos** [comment]
I ran the Base Tests locally and they went okay, @stianst could you restart the Model tests again?

File: testsuite/integration-arquillian/pom.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@
         <!--the version of shrinkwrap_resolver should align with the version in arquillian-bom-->
         <shrinkwrap-resolver.version>3.1.4</shrinkwrap-resolver.version>
         <selenium.version>3.14.0</selenium.version>
-        <arquillian-drone.version>2.5.2</arquillian-drone.version>
+        <arquillian-drone.version>2.5.4</arquillian-drone.version>
         <arquillian-graphene.version>2.3.2</arquillian-graphene.version>
```
Comment (mabartos) [inline testsuite/integration-arquillian/pom.xml:54]: Version `2.5.4`

File: testsuite/integration-arquillian/pom.xml [hunk 2]
```diff
@@ -50,7 +50,7 @@
...
         <arquillian-wildfly-container.version>2.1.1.Final</arquillian-wildfly-container.version>
```
Comment (mabartos) [inline testsuite/integration-arquillian/pom.xml:55]: Version `2.2.0.Final` or `3.0.1.Final`

**Review by mabartos — COMMENTED** [review]
Just one thing. It'd be great to update all Arquillian dependencies. WDYT? Some of them in comments and some here:

<arquillian-container-karaf.version>`2.2.2.Final`
<arquillian-infinispan-container.version>`1.2.0.Beta2`

**Comment by Aboullos** [comment]
> Just one thing. It'd be great to update all Arquillian dependencies. WDYT? Some of them in comments and some here:
> 
> <arquillian-container-karaf.version>`2.2.2.Final` <arquillian-infinispan-container.version>`1.2.0.Beta2`

I haven't check what those versions change, the change on the drone 2.5.2 to 2.5.4 is because on the 2.5.3 and 2.5.4 there are changes so that we can run tests with Edge.
Updating other versions could be done in another PR

**Review by mabartos — APPROVED** [review]
@Aboullos Ok. LGTM

**Pull request merged**